sensor_msgs/PointCloud2 scene_pc
sensor_msgs/PointCloud2 obj_top_pc
geometry_msgs/Point[] poly_pts
geometry_msgs/Point[] pot_pts
sensor_msgs/Image depth_image
float64[] location_weight
geometry_msgs/Quaternion[] heading
geometry_msgs/Point[] loc